exporter/stats/stackdriver/stackdriver.go

```diff
@@ -164,7 +165,7 @@ func (e *Exporter) makeReq(vds []*stats.ViewData) *monitoringpb.CreateTimeSeries
 		for _, row := range vd.Rows {
 			ts := &monitoringpb.TimeSeries{
 				Metric: &metricpb.Metric{
-					Type: path.Join("custom.googleapis.com", vd.View.Measure().Name()),
+					Type: path.Join(namespacedViewName(vd.View.Measure().Name())),
```

Comment:
In Java we use the name of View (instead of Measure) as the name of MetricDescriptor.

Reply:
Done